package.json

```diff
@@ -69,6 +69,7 @@
     "glob": "^4.5.3",
     "loader.js": "^4.2.3",
     "morgan": "^1.7.0",
+    "testem": "1.15.0",
```

Comment:
I think this is left over from one of my attempts to fix the constant rebuild issues I was having. I'm fine if we leave it in, but it can probably be done in a separate PR. If we have to revert, it'd be nice not to have the churn related to the testem version.